src/api/java/com/minecolonies/api/colony/IColony.java

```diff
@@ -300,6 +311,8 @@ default String getTeamName()
 
     void setColonyColor(TextFormatting color);
 
+    void setColonyFlag(ListNBT patterns);
```

Comment:
jdoc

Reply:
This is a setter, and few of the others have docs. I wouldn't want it to be alone ...